### ISSUE TITLE
Live Character Count/Limit

### ISSUE THREAD

**Issue by bamboochow** [whynotearth/Volkswagen-email-corporate-communications-CMS#287]
https://gyazo.com/2944f1c9a9578e62c8edccf8901da788

**Comment by paulchrisluke** [comment]
@mort3za @abdullahcalisir12 this should count characters not lines or words, and should look like vuetify's version above. It should also only be applied to the headline and description fields of memo writing and article writing.

**Comment by abdullahcalisir12** [comment]
@paulchrisluke at swagger I couldn't find max limit of subject and description at memo schema so I'm leaving them without live character counter

**Comment by paulchrisluke** [comment]
@ctyar can you provide endpoints for @abdullahcalisir12 so we can enable live character count? $

**Comment by ctyar** [comment]
Getting the text length from the API would create a very slow/laggy experience for the user (you need to make a request to the API for each keystroke)
We have to come up with a solution to calculate the text length without the markdown specific characters in the client-side and the API

**Comment by abdullahcalisir12** [comment]
@ctyar in article model there are max length attributes but in memo model there is no

![image](https://user-images.githubusercontent.com/29024217/83112178-7089af00-a0ce-11ea-9955-6039b0aa7cae.png)
![image](https://user-images.githubusercontent.com/29024217/83112202-7aabad80-a0ce-11ea-933e-01e743f9e5dd.png)


and for the markdown yes you're right it calculates markdown specific characters too so as you said er have to find a solution for that

**Comment by ctyar** [comment]
That's correct right now we have no limitation on the memo length. Do we need one?

**Comment by paulchrisluke** [comment]
No we do not need a character max length for memo sending

**Comment by abdullahcalisir12** [comment]
@paulchrisluke  okay then it's fine and no need to add character live counter to memo subject description 
![image](https://user-images.githubusercontent.com/29024217/83121728-5e623d80-a0db-11ea-961f-e2f109cf6d7f.png)
